app/models/account.rb

```diff
@@ -128,7 +130,7 @@ def set_omniauth_info_as_annotation
       em = Embed.new
       em.annotated = self
     end
-    em.embed_for_registration_account(self.user.omniauth_info)
+    em.embed_for_registration_account(self.omniauth_info)
```

Comment:
`Embed` is a terrible name for what this represents. `Metadata` would be much more appropriate. Is it feasible to rename the annotation and associated functions? @caiosba and @melsawy